karma.conf.js

```diff
@@ -85,6 +89,7 @@ module.exports = function(config) {
       "/tests/": "/base/tests/",
       "/spec/": "/base/tests/spec/",
       "/node_modules/": "/base/node_modules/",
+      "/builds/": "/base/builds/",
```

Comment:
Hmm - maybe I don't understand the karma testing strategy.  It builds the
built version as a first step.  Why does it do that if it is not going to
test it.  Also, while I agree that testing a static version is different
than testing the dynamic version, we don't /ship/ the dynamic version.  If
they test differently it is definitely a problem, but it is also a problem
if we do all our testing against something we don't ship.

What am I missing?

On Sat, Mar 5, 2016 at 3:11 AM, Marcos Caceres <notifications@github.com>
wrote:

> In karma.conf.js
> <https://github.com/w3c/respec/pull/611#discussion_r55118571>:
>
> > @@ -85,6 +89,7 @@ module.exports = function(config) {
> >        "/tests/": "/base/tests/",
> >        "/spec/": "/base/tests/spec/",
> >        "/node_modules/": "/base/node_modules/",
> > +      "/builds/": "/base/builds/",
>
> As above
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/w3c/respec/pull/611/files#r55118571>.
>



-- 
Shane McCarron
halindrome@gmail.com